packages/common/client-utils/package.json

```diff
@@ -90,7 +90,7 @@
 		"test:mocha:cjs": "mocha --recursive \"dist/test/mocha/**/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
 		"test:mocha:esm": "mocha --recursive \"lib/test/mocha/**/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
 		"tsc": "fluid-tsc commonjs --project ./tsconfig.cjs.json && copyfiles -f ./src/cjs/package.json ./dist",
-		"typetests:gen": "flub generate typetests --dir . -v --publicFallback",
+		"typetests:gen": "flub generate typetests --dir . -v --level public",
```

Comment:
Can you please note in description all of the things/places that were done specially besides a version bump and build?
(My build kept failing when I used local build-tools and I was so confused as I didn't see this change in the PR.)

Reply:
Point taken. I missed it myself when reviewing the commits (I try to keep manual changes in their own commits).